sparkle/platform/settings_help.py

```diff
@@ -201,6 +201,22 @@ def read_settings_ini(self: Settings, file_path: PurePath = DEFAULT_settings_pat
                     self.set_config_wallclock_time(value, state)
                     file_settings.remove_option(section, option)
 
+            section = "configuration"
+            option_names = ("cpu_time", "to_be_removed")
+            for option in option_names:
+                if file_settings.has_option(section, option):
+                    value = file_settings.getint(section, option)
+                    self.set_config_cpu_time(value, state)
+                    file_settings.remove_option(section, option)
+
+            section = "configuration"
+            option_names = ("solver_calls", "to_be_removed")
```

Comment:
If you want to define a tuple without a second value, simply put ("TEXT",)

Reply:
It was my understanding that the alternative name we used there was something that you were removing, so I wanted to make it clear that the second part will soon be obsolete